Submitted By: Mario Fetka (mario-fetka at gmx dot at)
Date: 2007-10-08
Initial Package Version: 1.0-pre20071008
Origin: me
Upstream Status: unknown
Description: prefix config entries with restartd
 
diff -Naur mod_restartd-1.0-pre20071008.orig/mod_restartd-2.2.c mod_restartd-1.0-pre20071008/mod_restartd-2.2.c
--- mod_restartd-1.0-pre20071008.orig/mod_restartd-2.2.c	2006-01-31 22:38:57.000000000 +0000
+++ mod_restartd-1.0-pre20071008/mod_restartd-2.2.c	2007-10-08 09:40:01.000000000 +0000
@@ -673,7 +673,7 @@
         if (cgid_req.ppid != parent_pid) {
             ap_log_error(APLOG_MARK, APLOG_CRIT, 0, main_server,
                          "CGI request received from wrong server instance; "
-                         "see ScriptSock directive");
+                         "see restartdScriptSock directive");
             close(sd2);
             continue;
         }
@@ -919,7 +919,7 @@
     conf->logname = ap_server_root_relative(cmd->pool, arg);
 
     if (!conf->logname) {
-        return apr_pstrcat(cmd->pool, "Invalid ScriptLog path ",
+        return apr_pstrcat(cmd->pool, "Invalid restartdScriptLog path ",
                            arg, NULL);
     }
     return NULL;
@@ -957,7 +957,7 @@
     sockname = ap_server_root_relative(cmd->pool, sockname);
 
     if (!sockname) {
-        return apr_pstrcat(cmd->pool, "Invalid ScriptSock path",
+        return apr_pstrcat(cmd->pool, "Invalid restartdScriptSock path",
                            arg, NULL);
     }
 
@@ -975,16 +975,16 @@
 
 static const command_rec cgid_cmds[] =
 {
-    AP_INIT_TAKE1("ScriptLog", set_scriptlog, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptLog", set_scriptlog, NULL, RSRC_CONF,
                   "the name of a log for script debugging info"),
-    AP_INIT_TAKE1("ScriptLogLength", set_scriptlog_length, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptLogLength", set_scriptlog_length, NULL, RSRC_CONF,
                   "the maximum length (in bytes) of the script debug log"),
-    AP_INIT_TAKE1("ScriptLogBuffer", set_scriptlog_buffer, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptLogBuffer", set_scriptlog_buffer, NULL, RSRC_CONF,
                   "the maximum size (in bytes) to record of a POST request"),
-    AP_INIT_TAKE1("ScriptSock", set_script_socket, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptSock", set_script_socket, NULL, RSRC_CONF,
                   "the name of the socket to use for communication with "
                   "the cgi daemon."),
-    AP_INIT_FLAG("RetainPerms", set_retainperms, NULL, OR_AUTHCFG,
+    AP_INIT_FLAG("restartdRetainPerms", set_retainperms, NULL, OR_AUTHCFG,
                   "Don't drop permissions when running the CGI."),
     {NULL}
 };
diff -Naur mod_restartd-1.0-pre20071008.orig/mod_restartd.c mod_restartd-1.0-pre20071008/mod_restartd.c
--- mod_restartd-1.0-pre20071008.orig/mod_restartd.c	2007-09-27 16:55:31.000000000 +0000
+++ mod_restartd-1.0-pre20071008/mod_restartd.c	2007-10-08 09:42:36.000000000 +0000
@@ -882,7 +882,7 @@
     conf->logname = ap_server_root_relative(cmd->pool, arg);
 
     if (!conf->logname) {
-        return apr_pstrcat(cmd->pool, "Invalid ScriptLog path ",
+        return apr_pstrcat(cmd->pool, "Invalid restartdScriptLog path ",
                            arg, NULL);
     }
     return NULL; 
@@ -917,7 +917,7 @@
     conf->sockname = ap_server_root_relative(cmd->pool, arg); 
 
     if (!conf->sockname) {
-        return apr_pstrcat(cmd->pool, "Invalid Scriptsock path ",
+        return apr_pstrcat(cmd->pool, "Invalid restartdScriptsock path ",
                            arg, NULL);
     }
 
@@ -935,16 +935,16 @@
 
 static const command_rec cgid_cmds[] = 
 { 
-    AP_INIT_TAKE1("ScriptLog", set_scriptlog, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptLog", set_scriptlog, NULL, RSRC_CONF,
                   "the name of a log for script debugging info"), 
-    AP_INIT_TAKE1("ScriptLogLength", set_scriptlog_length, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptLogLength", set_scriptlog_length, NULL, RSRC_CONF,
                   "the maximum length (in bytes) of the script debug log"), 
-    AP_INIT_TAKE1("ScriptLogBuffer", set_scriptlog_buffer, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptLogBuffer", set_scriptlog_buffer, NULL, RSRC_CONF,
                   "the maximum size (in bytes) to record of a POST request"), 
-    AP_INIT_TAKE1("Scriptsock", set_script_socket, NULL, RSRC_CONF,
+    AP_INIT_TAKE1("restartdScriptsock", set_script_socket, NULL, RSRC_CONF,
                   "the name of the socket to use for communication with "
                   "the cgi daemon."), 
-    AP_INIT_FLAG("RetainPerms", set_retainperms, NULL, OR_AUTHCFG,
+    AP_INIT_FLAG("restartdRetainPerms", set_retainperms, NULL, OR_AUTHCFG,
                   "Don't drop permissions when running the CGI."),
     {NULL} 
 }; 
